### PR TITLE
 Finished problems 41 and 42.

### PR DESCRIPTION
**Pull request by ikhat**
Possibly explained why the component function is linear in too much detail for you.

Also not sure if I'm too wordy in the last part of 42. 

One other thing: In your definition of the Image of a set before Problem 42, are you sure you don't want \vec x and \vec y switched (so it says L(x) = y)?

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -2943,14 +2943,50 @@ \section*{Linear Transformations}
...
+		\item Classify the following as linear transformations or not.
 			\begin{enumerate}
 				\item $\mathcal R$ from before (rotation counter-clockwise by $90^\circ$).
+					\begin{solution}
+						A linear transformation. We proved this in the previous problem.
+					\end{solution}
 				\item $W:\R^2\to\R^2$ where $W\mat{x\\y}=\mat{x^2\\y}$.
+					\begin{solution}
+						Not a linear transformation, since for example 
+						$W\left(2\mat{1\\0}\right) = \mat{4\\0} \neq 2W\mat{1\\0}$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: W(2[ 1 0]) = [4 0] != [2 0] = 2W[1 0]

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -2943,14 +2943,50 @@ \section*{Linear Transformations}
...
 					$\mathcal P\mat{x\\y}=\Comp_{\vec u}\mat{x\\y}$ and 
 					$\vec u=\mat{2\\3}$.
+					\begin{solution}
+						A linear transformation.
+
+						We found a general formula for $\Comp_{\vec u}$ in a previous
+						exercise:
+						\[
+							\Comp_{\vec u}\vec x
+							=\frac{\vec u\cdot\vec x}{\norm{\vec u}^2}\vec u
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: u dot u on the bottom

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -2974,24 +3010,51 @@ \section*{Linear Transformations}
...
+				$W(C) = C$.
+
+				$T(C) = \Set*{\mat{2\\0},\mat{3\\0},\mat{2\\1},\mat{3\\1}}$.
+			\end{solution}
+		\item Draw $\mathcal R(S)$, $T(S)$, and $\mathcal P(S)$ (where $\mathcal R$,
+			$T$, and $\mathcal P$ are from the previous question).
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item Suppose that $\ell=\Set{\text{all convex combinations of }\vec a\text{ and }\vec b}$
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Rephrase: Let \ell ... be the line segment and let A be a linear transformation.

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -2974,24 +3010,51 @@ \section*{Linear Transformations}
...
+			is a line segment with endpoints $\vec a$ and $\vec b$ and $A$ is a 
+			linear transformation. Must $A(\ell)$ be a line segment? 
+			What are its endpoints?
+			\begin{solution}
+				Yes, $A(\ell)$ must be a line segment.
+				Its endpoints are $A(\vec a)$ and $A(\vec b)$.
+
+				If $\vec x=\alpha_1\vec a+\alpha_2\vec b$ is a convex combination
+				from $\ell$, then $A(\vec x)=\alpha_1A(\vec a)+\alpha_2A(\vec b)$,
+				which is a convex combination of $A(\vec a)$ and $A(\vec b)$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Discuss the degenerate case.

File: problemsets-mat223/linearalbegra.tex (hunk 3 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Not entirely sure what you mean here. The problem defines \ell and A so I ordinarily wouldn't say "Let..." again. But are you imagining something like this:

Let \ell = {all convex linear combinations of a and b} be the line segment and A be a linear transformation. By the linearity of A

A(\alpha_1 a + \alpha_2 b) = \alpha_1 A(a) + \alpha_2 A(b).

If \alpha_1 + \alpha_2 = 1, then the linear combination on the right is also convex, and so A(\ell) is the set of convex combinations of A(a) and A(b). This is precisely the straight line segment joining A(a) and A(b).

File: problemsets-mat223/linearalbegra.tex (hunk 4 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Do you mean the case in which a=b?

File: problemsets-mat223/linearalbegra.tex (hunk 3 above)
Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: Yep! That's what I was thinking. Though, I meant to suggest that you rephrase the problem as "Let ...". You don't need to say "let" in your solution.

File: problemsets-mat223/linearalbegra.tex (hunk 4 above)
Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: Well, `A` could be the zero transformation, so `A(a)` to `A(b)` would be a degenerate segment, for example.

Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Oh, I see. I'll make a note about what happens if A(a) = A(b), such as with the zero transformation.
